src/platform/qt/AboutDialog.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2013-2015 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include <QDialog>
+
+#include "ui_AboutDialog.h"
+
+namespace QGBA {
+
+class AboutDialog : public QDialog {
+Q_OBJECT
+
+public:
+	AboutDialog(QDialog* parent = nullptr);
+
+private:
+	Ui::AboutDialog m_ui;
+};
+
+}
```

Comment:
Make sure there's a newline at the end of the file; some editors get angry if it's missing.